zip-0227.rst

```diff
@@ -1,8 +1,359 @@
+- the `asset_desc` is a general byte string in order to allow for a wide range of information type to be included that may be associated with the assets. Some are:
+    - links for storage such as for NFTs
+    - other metadata for native assets
+    - bridging information for wrapped assets (chain of origin, issuer name, etc)
+    - information to be committed by the issuer, though not enforceable by the protocol
+
+TxId Digest
+===========
+A new issuance transaction digest algorithm is defined that constructs the identifier for an issuance transaction. Each branch of the tree of hashes will correspond to a specific subset of transaction data. The overall structure of the hash is as follows; each name referenced here will be described in detail below::
+
+    issuance_txid_digest
+    ├── issueActions
+    └── issuerVerificationKey
+
+In the specification below, nodes of the tree are presented in depth-first order.
+
+issuance_txid_digest
+--------------------
+A BLAKE2b-256 hash of the following values ::
+
+   T.1: issueActions            (field encoding bytes)
+   T.2: issuerVerificationKey   (field encoding bytes)
+
+The personalization field of this hash is set to::
+
+  "ZTxIdOrcZSAIssue"
+
+T.1 issueActions
+````````````````
+A byte encoding of Issue Action information for all Issue Actions belonging to the transaction. For each Action, the following elements are included in the hash::
+
+   T.1a: notes                   (field encoding bytes)
+   T.1b: assetDescription        (field encoding bytes)
+   T.1c: isFinalized             (1 byte)
+
+
+T.1a: notes
+'''''''''''
+A byte encoding of Note information for all Notes belonging to the Action. For each Note, the following elements are included in the hash::
+
+   T.1a.1: recipient                    (field encoding bytes)
+   T.1a.2: value                        (field encoding bytes)
+   T.1a.3: asset                        (field encoding bytes)
+   T.1a.4: rho                          (field encoding bytes)
+   T.1a.5: rseed                        (field encoding bytes)
+
+
+T.1a.1: recipient
+.................
+Raw Address encoded as specified in [Zcash Protocol Spec § 5.6.4.2: Orchard Raw Payment Addresses].
+
+T.1a.2: value
+.............
+Note value encoded as little-endian 8-byte representation of u64 raw value.
+
+T.1a.3: asset
+.............
+Asset ID encoded as 32-byte representation of Pallas point.
+
+T.1a.4: rho
+...........
+Nullifier encoded as 32-byte representation of Pallas point.
+
+T.1a.5: rseed
+.............
+The ZIP 212 32-byte seed randomness for a note.
+
+T.1b: assetDescription
+''''''''''''''''''''''
+UTF-8 encoding of the asset description string.
+
+T.1c: isFinalized
+'''''''''''''''''
+1-byte representation of a boolean flag that is set to 'true' if the asset type was finalized in this action and 'false' otherwise. 'True' is represented as 1, 'false' as 0.
```

Comment:
You could copy-paste-edit the `flagsOrchard` description here:
https://github.com/zcash/zips/blob/main/zip-0225.rst#transaction-format

Reply:
Done.